cli/README.md

```diff
@@ -6,21 +6,18 @@ Once a set of queues are populated the consumers can be started, updated and the
 stopped cleanly.
 
 ## Prerequisites
-
-`PIXL CLI` requires Python version 3.10.
-
-Running the tests requires [docker](https://docs.docker.com/get-docker/) to be installed.
-
-## Installation
-
-We recommend installing in a project specific virtual environment created using a environment
+* Python version 3.11 (matching python versions in [pixl-ci](../.github/workflows/main.yml) and [dev](../docs/setup/developer.md#installation-of-pixl-modules)).
+* [Docker](https://docs.docker.com/get-docker/) with version `>=27.0.3`
+* [Docker Compose](https://docs.docker.com/compose/install/#installation-scenarios) with version `>=v2.28.1-desktop.1`
+* We recommend installing PIXL project in specific virtual environment using a environment
 management tool such as [conda](https://docs.conda.io/en/latest/) or
-[virtualenv](https://virtualenv.pypa.io/en/latest/).
-
-Then install in editable mode by running
+[virtualenv](https://virtualenv.pypa.io/en/latest/). 
+See detailed instructions [here](../docs/setup/developer.md#setting-up-python-virtual-environment)
 
+## Installation
+Activate your python virtual environment and install `PIXL` project in editable mode by running
 ```bash
-python -m pip install -e ../pixl_core/ -e .
+pip install -e ../pixl_core -e .
```

Comment:
I'm not sure if this was intentional but should probably be `python -m`

Reply:
sorted out here e30ee053808506446244759de74d6fe4dee229eb